### DISCUSSION TITLE
Unable to delete faulty custom script in v3.5.0

### DISCUSSION THREAD

**Discussion by philket** [netbox-community/netbox#12451] [Help Wanted!]
Hey,
I really like the changes to they way custom scripts are implemented in 3.5.0, but it still seems a little buggy and not well documented.
One issue that I found, is that when I upload a script that has some kind of python error in it, I get an error page when I click on _Customization_ -> _Scripts_ in the WebGUI. This is the same as in previous versions. The problem is though, that I am now unable to delete this faulty script or really do anything to solve the error, since I only get this error page. Or am I overlooking something?

**Comment by bitcollector1** [comment]
I ran into same issue.  

I worked around it by deleting the script via CLI and the adding a working version back in via CLI.  

I think you can use any script with the same name, as long as it works. 

Then you can delete the script from within the UI itself. 

It's a hack but it worked for me :) 

Hope it helps!

**Comment by philket** [comment]
This seems resolved with #12456
